src/include/sof/schedule/edf_schedule.h

```diff
@@ -35,4 +35,9 @@ struct edf_task_pdata {
 
 int scheduler_init_edf(void);
 
+int schedule_edf_task_init(struct task *task, uint16_t priority,
```

Comment:
Better to make these helpers (as discussed above) and use "edf" as the suffix (to be consistent with the other APIs).

Reply:
@lgirdwood I would like to clarify this helper usage. Do you mean we should first call schedule_task_init_edf/ll and after that generic schedule_task_init?

Reply:
yes, this keeps the core APIs consistent whilst allowing some scheduler specific helper APIs.

Reply:
But schedule_task_init_edf/ll call generic schedule_task_init inside them. I don't think calling it outside of the specific init functions is good.

Reply:
Sorry, I meant call inside which is what you are doing.